crates/bevy_winit/src/lib.rs

```diff
@@ -664,16 +669,22 @@ pub fn winit_runner(mut app: App) {
                 runner_state.is_active = false;
                 #[cfg(target_os = "android")]
                 {
-                    // Android sending this event invalidates all render surfaces.
-                    // TODO
-                    // Upon resume, check if the new render surfaces are compatible with the
-                    // existing render device. If not (which should basically never happen),
-                    // then try to rebuild the renderer.
-                    *control_flow = ControlFlow::Exit;
+                    let mut query = app.world.query::<(Entity, &Window)>();
+                    let (entity, window) = query.single(&app.world);
+                    primary_window = Some(window.clone());
+                    app.world.despawn(entity);
```

Comment:
The despawn/respawn behavior here will result in the Entity ID of each window,and the additional components on the Window to not be retained. Is this acceptable? If it is, we should document this clearly in a visible location.

Reply:
I think it's acceptable. I added some documentation on the example, which should be the first place people trying to run on mobile should look

Reply:
i think current window should be modified in-place if possible